src/main/java/org/fogbowcloud/arrebol/ArrebolController.java

```diff
@@ -148,8 +182,8 @@ public String addJob(Job job) {
 
     public String stopJob(Job job) {
 
-        for(Task task : job.getTasks()){
-         //
+        for (Task task : job.getTasks()) {
```

Comment:
why don't we remove this dead code?

Reply:
also, add an "//still unsuportted"  comment

Reply:
Done.